src/Illuminate/Foundation/Testing/TestCase.php

```diff
@@ -103,39 +109,37 @@ protected function refreshApplication()
     }
 
     /**
-     * Boot the testing helper traits.
+     * Boot the testing helper methods.
      *
-     * @return array
+     * @return $this
      */
-    protected function setUpTraits()
+    protected function setUpTestHelpers()
```

Comment:
I see this is targeting 7.x, but nevertheless this can be a potential huge breaking change. I for one use `setupTraits` in my own code, calling this parent and expecting the returned value (array of traits) because my code does the same check.

Reply:
To not be a breaking change, I can rename the method to `setUpTraits` again. I just renamed it, because with my changes, the method name wouldn't make no sense anymore.
But I could also change the method to do this:
```php
if (isset($uses[RefreshDatabase::class]) || $this instanceof DatabaseRefreshable) {
    $this->refreshDatabase();
}
```
So only using the helper traits would work too.

Reply:
If you try to maintain compatibility, you can't change the return type and still need to return the `$uses` array.

Reply:
Thank you for the help.
But I will close this PR and open a new for the 6.x branch because the changes are backwards compatible